rpmfusion-free-rawhide.repo

```diff
@@ -3,7 +3,8 @@ name=RPM Fusion for Fedora Rawhide - Free
 #baseurl=http://download1.rpmfusion.org/free/fedora/development/$basearch/os/
 metalink=https://mirrors.rpmfusion.org/metalink?repo=free-fedora-rawhide&arch=$basearch
 enabled=1
-gpgcheck=0
+gpgcheck=1
```

Comment:
repo_gpgcheck is a different things.
It's still not used on fedora and is discussed at https://bugzilla.rpmfusion.org/569

I don't know the requirement for repo_gpgsign on the infra side yet.